torch/quantization/fx/quantize.py

```diff
@@ -374,6 +374,28 @@ def load_arg(a):
                 graph_inputs.append(node.name)
 
         get_new_observer_name = get_new_attr_name_with_prefix('activation_post_process_')
+        model_device = assert_and_get_unique_device(model)
```

Comment:
Can we pass in the model_device or compute it inside the function directly?

Reply:
why? isn't it the same? in that case `model` will be captured in the closure..